.github/workflows/argus-docker-build.yaml

```diff
@@ -52,14 +52,21 @@ jobs:
     permissions:
       id-token: write
       contents: read
+      pull-requests: read
     steps:
-      - uses: chanzuckerberg/github-actions/.github/actions/argus-builder/build-prep@c210eb8dba1aeb3ccf02186ad89c3c85f6fc4da7
+      - uses: 8BitJonny/gh-get-current-pr@3.0.0
```

Comment:
does this work on `push` and on `pull_request`?

Reply:
From the docs: 

`Inside a running github action you can't always get the information whether you are currently in a PR or not. If for example the trigger event is pull_request you have that information but not when your trigger event is push.`